diff --git a/es/generate.d.ts b/es/generate.d.ts
index 5e7eec19327c4012edf022fee947d330527b5cb6..2bc61353c22b73e563c2ca7c80743069f3b3cdcc 100644
--- a/es/generate.d.ts
+++ b/es/generate.d.ts
@@ -1,4 +1,4 @@
-interface Opts {
+export interface Opts {
     theme?: 'dark' | 'default';
     backgroundColor?: string;
 }
diff --git a/lib/generate.d.ts b/lib/generate.d.ts
index 5e7eec19327c4012edf022fee947d330527b5cb6..2bc61353c22b73e563c2ca7c80743069f3b3cdcc 100644
--- a/lib/generate.d.ts
+++ b/lib/generate.d.ts
@@ -1,4 +1,4 @@
-interface Opts {
+export interface Opts {
     theme?: 'dark' | 'default';
     backgroundColor?: string;
 }
